edge-modules/load-gen/load-gen.csproj

```diff
@@ -35,6 +35,7 @@
   <ItemGroup>
     <ProjectReference Include="..\..\edge-util\src\Microsoft.Azure.Devices.Edge.Util\Microsoft.Azure.Devices.Edge.Util.csproj" />
     <ProjectReference Include="..\ModuleLib\Microsoft.Azure.Devices.Edge.ModuleUtil.csproj" />
+    <ProjectReference Include="..\..\test\modules\TestResultCoordinator\TestResultCoordinator.csproj" />
```

Comment:
remove this once TestResultCoordinatorClient is relocated to Microsoft.Azure.Devices.Edge.ModuleUtil project.